app/src/main/res/values/dimens.xml

```diff
@@ -46,4 +46,7 @@
     <dimen name="dp_26">26dp</dimen>
     <dimen name="dp_350">350dp</dimen>
     <dimen name="sp_30">30sp</dimen>
+    <dimen name="dp_36">36dp</dimen>
```

Comment:
WARNING: [UnusedResources] The resource `R.dimen.dp_36` appears to be unused
 Unused resources make applications larger and slow down builds.